playbooks/playbook-desktops.yml

```diff
@@ -55,4 +53,4 @@
         name: "lnls-ans-role-glusterfs"
       when: "'control_room_all' in group_names"
     - include_role:
-        name: "lnls-ans-role-nfsclient"
+        name: "lnls-ans-role-nfsclient"
```

Comment:
no newline at end of file

Reply:
Newline was added.

Reply:
@rbatistacardoso , my 'changed files' still shows a missing newline at end of file. I took the liberty to push commit to fix it, ok?